edgecast/resources/cps/certificate.go

```diff
@@ -422,8 +422,53 @@ func ResourceCertificateRead(ctx context.Context,
 	d *schema.ResourceData,
 	m interface{},
 ) diag.Diagnostics {
-	// Not Yet Implemented
 	var diags diag.Diagnostics
+
+	config := m.(**api.ClientConfig)
+	cpsService, err := buildCPSService(**config)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	params := certificate.NewCertificateGetParams()
+	n, err := strconv.ParseInt(d.Get("id").(string), 10, 64)
+	if err == nil {
+		fmt.Printf("%d of type %T", n, n)
+	}
+	params.ID = n
+	resp, err := cpsService.Certificate.CertificateGet(params)
+	if err != nil {
+		d.SetId("")
+		return diag.FromErr(err)
+	}
+
+	log.Printf("[INFO] Retrieved certificate: %# v", pretty.Formatter(resp))
+
+	d.SetId(strconv.FormatInt(resp.ID, 10))
+	d.Set("certificate_label", resp.CertificateLabel)
+	d.Set("description", resp.Description)
+	d.Set("last_modified", resp.LastModified.String())
```

Comment:
This is not just for CPS, but we need to keep some common logic to convert these dates to strings using a format we specify. If the underlying format of `String()` changes it could cause diff issues for users. Currently it uses RFC3339Millis:
https://github.com/go-openapi/strfmt/blob/a239ff1fdb33f31ad470c61555006bb09260aa59/time.go#L122